common/src/Utilities/Windows/WinMisc.cpp

```diff
@@ -48,133 +48,24 @@ u64 GetPhysicalMemory()
     return status.ullTotalPhys;
 }
 
-typedef void(WINAPI *PGNSI)(LPSYSTEM_INFO);
-typedef BOOL(WINAPI *PGPI)(DWORD, DWORD, DWORD, DWORD, PDWORD);
-
-// Win 10 SDK
-#ifndef PRODUCT_CORE_N
-#define PRODUCT_CORE_N 0x00000062
-#endif
-#ifndef PRODUCT_CORE
-#define PRODUCT_CORE 0x00000065
-#endif
-#ifndef PRODUCT_PROFESSIONAL_WMC
-#define PRODUCT_PROFESSIONAL_WMC 0x00000067
-#endif
-#ifndef PRODUCT_EDUCATION
-#define PRODUCT_EDUCATION 0x00000079
-#endif
-#ifndef PRODUCT_EDUCATION_N
-#define PRODUCT_EDUCATION_N 0x0000007A
-#endif
-#ifndef PRODUCT_ENTERPRISE_S
-#define PRODUCT_ENTERPRISE_S 0x0000007D
-#endif
-#ifndef PRODUCT_ENTERPRISE_S_N
-#define PRODUCT_ENTERPRISE_S_N 0x0000007E
-#endif
-
-// Calculates the Windows OS Version and install information, and returns it as a
+// Calculates the Windows OS Version and processor architecture, and returns it as a
 // human-readable string. :)
-// (Handy function borrowed from Microsoft's MSDN Online, and reformatted to use wxString.)
 wxString GetOSVersionString()
 {
     wxString retval;
 
-    OSVERSIONINFOEX osvi;
     SYSTEM_INFO si;
-    PGNSI pGNSI;
-    PGPI pGPI;
-    BOOL bOsVersionInfoEx;
-    DWORD dwType;
-
-    memzero(si);
-    memzero(osvi);
-
-    osvi.dwOSVersionInfoSize = sizeof(OSVERSIONINFOEX);
-// GetVersionEx is deprecated, but we don't use it for version checking anyways
-#pragma warning(suppress : 4996)
-    if (!(bOsVersionInfoEx = GetVersionEx((OSVERSIONINFO *)&osvi)))
-        return L"GetVersionEx Error!";
-
-    // Call GetNativeSystemInfo if supported or GetSystemInfo otherwise.
-
-    pGNSI = (PGNSI)GetProcAddress(GetModuleHandle(L"kernel32.dll"), "GetNativeSystemInfo");
-    if (NULL != pGNSI)
-        pGNSI(&si);
-    else
-        GetSystemInfo(&si);
+	GetNativeSystemInfo(&si);
 
     if (!IsWindows8Point1OrGreater())
         return L"Unsupported Operating System!";
 
     retval += L"Microsoft ";
 
-    // Test for the specific product.
-
     if (IsWindows10OrGreater())
-        retval += IsWindowsServer() ? L"Windows Server 2016 " : L"Windows 10 ";
+        retval += IsWindowsServer() ? L"Windows Server 2016 " : L"Windows 10";
```

Comment:
I believe the spacing after Windows 10 string is intentional.

Reply:
It would be "Windows 10 , 64 bit" then

Reply:
`It would be "Windows 10 , 64 bit" then`
Your change is fine then.
`I did just now notice that I didn't do it for the other strings....`
Yes need to do the other 2.